pkgs/by-name/ap/apt-mirror/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, perl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apt-mirror";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "apt-mirror";
+    repo = "apt-mirror";
+    rev = finalAttrs.version;
+    hash = "sha256-GNsyXP/O56Y+8QhoSfMm+ig5lK/K3Cm085jxRt9ZRmI=";
+  };
+
+  nativeBuildInputs = [ perl ];
+
+  makeFlags = ["DESTDIR=$(out)" "PREFIX=''"];
```

```suggestion
  makeFlags = [ 
    "PREFIX=$(out)"
    "BASE_PATH=$(out)/var/spool/apt-mirror"
  ];
```
P.S.: what does this `var/spool/` do?

---

The Makefile [uses DESTDIR before BASE_PATH](https://github.com/apt-mirror/apt-mirror/blob/master/Makefile#L15), so it should be fine.

---

You need to use DESTDIR, as it is used everywhere (you'll get errors without it).

---

The `var/spool/` seems to be a directory for data which awaits further processing.